frontend/src/components/dashboard/SelectFilter.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <v-menu offset-y :close-on-content-click="!multiple">
+    <template #activator="{ on, attrs }">
+      <v-chip label outlined :color="active ? 'primary' : null" v-bind="attrs" v-on="on">
+        <span class="d-none d-sm-inline">{{
+          labelValue ? `${label}: ${labelValue}` : label
+        }}</span>
+        <span class="d-sm-none">{{ labelValue || label }}</span>
+        <v-icon right>mdi-chevron-down</v-icon>
+      </v-chip>
+    </template>
+    <v-list dense>
+      <v-list-item dense color="primary" @click.prevent="clear()">
+        <v-list-item-title>{{
+          $tc('components.dashboard.selectFilter.all')
+        }}</v-list-item-title>
+      </v-list-item>
+      <v-list-item
+        v-for="(item, self) in processedItems"
+        :key="self"
+        dense
+        :input-value="item.selected"
+        color="primary"
+        @click.prevent="toggle(item.value)"
+      >
+        <v-list-item-title>
+          <slot name="item" v-bind="{ item, self }">{{ item.text }} </slot>
+        </v-list-item-title>
+        <v-list-item-action v-if="multiple">
+          <v-checkbox v-model="item.selected" dense />
+        </v-list-item-action>
+      </v-list-item>
+    </v-list>
+  </v-menu>
+</template>
+
+<script>
+import { get, keyBy } from 'lodash'
+
+export default {
+  name: 'SelectFilter',
+  props: {
+    label: { type: String, required: true },
+    multiple: { type: Boolean, default: false },
+    value: { type: [Array, String], default: null },
+    items: { type: Object, default: () => ({}) },
+    displayField: { type: [String, Function], required: true },
+    valueField: { type: String, default: '_meta.self' },
+  },
+  computed: {
+    active() {
+      return this.value?.length > 0
+    },
+    processedItems() {
+      return keyBy(
+        Object.values(this.items).map((item) => {
+          const text = this.displayValue(item)
+          const value = get(item, this.valueField)
+          const selected = this.multiple
+            ? this.value?.includes(value)
+            : this.value === value
+          return { text, value, selected }
+        }),
+        'value'
+      )
+    },
+    labelValue() {
+      return this.multiple
+        ? (this.value || []).map((item) => this.processedItems[item].text).join(', ')
+        : this.processedItems[this.value]?.text
```

Comment:
Ah i didnt think of that, because there was no multiselect OR filter in the mockup.

This kind of sentence building is not trivial in other languages (oxford comma is the simplest of the special cases). Ruby on rails has a special helper for it which handles different languages: https://apidock.com/rails/Array/to_sentence
Are you sure the users will notice the difference? For the n:1 case it's clear that it must be OR, since AND does not make sense. For AND, maybe we can add a result counter to each unchecked option so users will notice that the number of results will go down with each additional selected user..?

Reply:
I'm fine with comma separated values for the first shot

Reply:
Okay. But I think it is very important to implement this (at least for the AND filter) just because it has a different meaning. If you don't do it in this PR please open an Issue.

Reply:
I found https://caniuse.com/?search=ListFormat which is apparently the JS equivalent of the Ruby on Rails `to_sentence` function. Browser support seems to be okay, and it can handle languages out of the box.

Reply:
Very cool! 🤩